src/libs/Navigation/OnyxTabNavigator.tsx

```diff
@@ -107,13 +103,17 @@ function OnyxTabNavigator({
         onActiveTabFocusTrapContainerElementChanged?.(selectedTab ? focusTrapContainerElementMapping[selectedTab] : null);
     }, [selectedTab, focusTrapContainerElementMapping, onActiveTabFocusTrapContainerElementChanged]);
 
+    if (onyxFetchStatus === CONST.ONYX_FETCH_STATUS.LOADING) {
+        return null;
+    }
+
```

Comment:
Excuse @dominictb, why do we need this?

Reply:
Thank you for clarify @dominictb, Im still testing. For now, what do you think about removing `onyxFetchStatus === CONST.ONYX_FETCH_STATUS.LOADING` by `if (isLoadingOnyxValue(onyxFetchStatus)) {`? To simplify.

Reply:
The best approach is to use the `isLoadingOnyxValue()` helper as it was designed for this kind of use case, and then we can also. remove `CONST.ONYX_FETCH_STATUS`.

Reply:
Perfect thanks for chiming in! @dominictb lets update this please